--- SettingsTabWine.cs	2022-08-09 15:29:39.146847216 -0500
+++ SettingsTabWine.cs	2022-08-09 15:30:27.344810958 -0500
@@ -24,19 +24,17 @@
                 CheckVisibility = () => startupTypeSetting.Value == WineStartupType.Custom
             },
 
-#if !FLATPAK
             new SettingsEntry<bool>("Enable Feral's GameMode", "Enable launching with Feral Interactive's GameMode CPU optimizations.", () => Program.Config.GameModeEnabled ?? true, b => Program.Config.GameModeEnabled = b)
             {
                 CheckVisibility = () => RuntimeInformation.IsOSPlatform(OSPlatform.Linux),
                 CheckValidity = b =>
                 {
-                    if (b == true && !File.Exists("/usr/lib/libgamemodeauto.so.0"))
+                    if (b == true && !File.Exists("/app/lib/libgamemodeauto.so.0"))
                         return "GameMode not detected.";
 
                     return null;
                 }
             },
-#endif
 
             new SettingsEntry<bool>("Enable DXVK ASYNC", "Enable DXVK ASYNC patch.", () => Program.Config.DxvkAsyncEnabled ?? true, b => Program.Config.DxvkAsyncEnabled = b),
             new SettingsEntry<bool>("Enable ESync", "Enable eventfd-based synchronization.", () => Program.Config.ESyncEnabled ?? true, b => Program.Config.ESyncEnabled = b),
